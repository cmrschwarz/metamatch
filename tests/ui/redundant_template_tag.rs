use metamatch::template;

fn main() {
    let res = template! {
        [<template>]
        42
        [</template>]
    };

    assert_eq!(res, 10);
}
