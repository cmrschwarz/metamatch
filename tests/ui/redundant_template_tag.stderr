error: redundant template tag, already inside a template
 --> tests/ui/redundant_template_tag.rs:5:11
  |
5 |         [<template>]
  |           ^^^^^^^^
